test/integration2/query/select/selects.spec.js

```diff
@@ -993,7 +993,7 @@ describe('Selects', function () {
         try {
           await knex.transaction((trx) => {
             // select all from two test tables and lock only one table
-            return trx('test_default_table')
+            return trx.withSchema('public').from('test_default_table')
```

Comment:
I'm not sure you'd say `.withSchema('public')` in a real query but it lets me exercise the behavior in question without having to learn enough about knex's test environment to stand up a discrete schema 😁